String/string.cpp

```diff
@@ -57,10 +57,7 @@ char String::charAt(int index) {
 }
 
 int String::compareTo(String anotherString){
-	if(size > anotherString.size){
-		return size-anotherString.size;
-	}
-	else if(size < anotherString.size){
+	if(size != anotherString.size){
 		return size-anotherString.size;
 	}
```

Comment:
you perform the same action for 2 different checks, can unionize to a single check:
either `(size > anotherString.size || size < anotherString.size)` , or simply `(size != anotherString.size)`